torch/_tensor_str.py

```diff
@@ -266,6 +266,20 @@ def _str(self):
         if values.numel() == 0:
             values_str += ', size=' + str(tuple(values.shape))
         tensor_str = indices_prefix + indices_str + '),\n' + ' ' * indent + values_prefix + values_str + ')'
+    elif self.is_quantized:
+        suffixes.append('size=' + str(tuple(self.shape)))
+        suffixes.append('dtype=' + str(self.dtype))
```

Comment:
nit: this should have a `if not has_default_dtype:` even though that can't currently be false.